With Hermes enabled:

```diff
diff --git a/example/macos/Podfile b/example/macos/Podfile
index 2266ea3..44b1ec3 100644
--- a/example/macos/Podfile
+++ b/example/macos/Podfile
@@ -2,7 +2,7 @@ require_relative '../node_modules/react-native-test-app/macos/test_app'

 workspace 'Example.xcworkspace'

-use_test_app! :hermes_enabled => false do |target|
+use_test_app! :hermes_enabled => true do |target|
   target.tests do
     pod 'Example-Tests', :path => '..'
   end
```

I am now getting linker errors:

```
Undefined symbols for architecture x86_64:
  "folly::exception_ptr_get_type(std::exception_ptr const&)", referenced from:
      folly::exception_wrapper::ExceptionPtr::type_(folly::exception_wrapper const*) in libRCT-Folly.a(ExceptionWrapper.o)
      folly::exceptionStr(std::exception_ptr const&) in libRCT-Folly.a(ExceptionString.o)
  "folly::exception_ptr_get_object(std::exception_ptr const&, std::type_info const*)", referenced from:
      std::exception* folly::exception_ptr_get_object<std::exception>(std::exception_ptr const&) in libReact-hermes.a(Connection.o)
      std::exception* folly::exception_ptr_get_object<std::exception>(std::exception_ptr const&) in libRCT-Folly.a(ExceptionWrapper.o)
      std::exception* folly::exception_ptr_get_object<std::exception>(std::exception_ptr const&) in libRCT-Folly.a(ExceptionString.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Edit: Fixed upstream: https://github.com/microsoft/react-native-macos/pull/982

--- a/example/package.json
+++ b/example/package.json
@@ -16,10 +16,10 @@
     "windows": "react-native run-windows --sln windows/Example.sln"
   },
   "peerDependencies": {
-    "react": "~16.8.6 || ~16.9.0 || ~16.11.0 || ~16.13.1 || ~17.0.1",
-    "react-native": "^0.0.0-0 || ^0.60.6 || ^0.61.5 || ^0.62.2 || ^0.63.2 || ^0.64.0 || 1000.0.0",
-    "react-native-macos": "^0.60.0 || ^0.61.0 || ^0.62.0 || ^0.63.0 || ^0.64.0",
-    "react-native-windows": "^0.62.0 || ^0.63.0 || ^0.64.0"
+    "react": "~16.8.6 || ~16.9.0 || ~16.11.0 || ~16.13.1 || ~17.0.1 || ~17.0.2",
+    "react-native": "^0.0.0-0 || 0.60 - 0.67 || 1000.0.0",
+    "react-native-macos": "^0.0.0-0 || 0.60 - 0.66",
+    "react-native-windows": "^0.0.0-0 || 0.62 - 0.67"
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",

